example-swift/example-swift/ViewController.swift

```diff
@@ -53,12 +54,14 @@ class ViewController: UITableViewController {
         let cvc = fieldCardCvc.text ?? ""
         let month = fieldCardMonth.text ?? ""
         let year = fieldCardYear.text ?? ""
-        print("input number=\(number), cvc=\(cvc), month=\(month), year=\(year)")
+        let name = filedCardName.text
```

Comment:
Please provide a default value for name to prevent warning coming out from the print statement.